primitives/ethereum-poa/src/lib.rs

```diff
@@ -303,6 +324,7 @@ impl SealedEmptyStep {
 	}
 
 	/// Returns rlp for the vector of empty steps (we only do encoding in tests).
+	// TODO: Do we really need an entire feature flag for one method?
```

Comment:
I added a few more things under this feature flag, so it might be sticking around